soc/xtensa/intel_adsp/common/mp_cavs.c

```diff
@@ -63,16 +63,6 @@ void soc_start_core(int cpu_num)
 {
 	uint32_t curr_cpu = prid();
 
-#ifdef CONFIG_SOC_SERIES_INTEL_CAVS_V15
```

Comment:
this change as about cavs18, why are we removing this section related to v15?

Reply:
It's being moved to below the write to PWRCTL.  It used to apply only to 1.5, not it applies to 1.5 and 1.8  (which is expressed as "not 2.5").